tables/v1/user_oauth2_tokens.sql

```diff
@@ -3,7 +3,8 @@ CREATE TABLE v1.user_oauth2_tokens (
     integration TEXT NOT NULL REFERENCES v1.oauth2_integrations(name) ON DELETE RESTRICT ON UPDATE CASCADE,
     external_id TEXT NOT NULL,
     access_token TEXT NOT NULL,
-    refresh_token TEXT NOT NULL,
+    refresh_token TEXT,
+    oidc_token TEXT,
```

Comment:
What about calling this `id_token` so that it matches the OIDC spec name? Especially since `access_token` and `refresh_token` came from the OAuth 2 spec.

Reply:
Yeah good point